### DISCUSSION TITLE
saiblo平台账户名与github账户名不一致的问题

### DISCUSSION THREAD

**Discussion by DM0U** [physics-data/faq#50] [Q&A] [answered]
在开课前已经在saiblo平台注册过账号，故账号与github名不一致。（而saiblo平台中并不允许修改用户名）
希望老师能帮忙手动建立映射关系。

saiblo平台账号：Pendragon
github账号：DM0U

**Comment by Harry-Chen** [comment, marked as answer]
感谢告知
